opendatahub-io/odh-dashboard
# [WIP] Pipeline cr view

closes: https://github.com/opendatahub-io/odh-dashboard/issues/1061

## Description


![Screenshot 2023-05-03 at 8 55 13 AM](https://user-images.githubusercontent.com/12587674/235950080-c1afc68b-36e4-4a9b-883f-a824a752c82c.png)

Added view

## How Has This Been Tested?



1. create a new pipeline server with both object storage and a new external database 
2. Pipeline server action > View pipeline server configuration
3. make sure all values are correct and secret values are correct

## Test Impact


n/a

## Request review criteria:





cc @andrewballantyne @kywalker-rh
